### PR TITLE
Feature/r17 roadmap

### PR DESCRIPTION
**Pull request by jeffvg**
Changes proposed in this Pull Request:
- Add deferred R16 work to R17
- Add React & Angular login and reg workflow data
-

### CODE REVIEW

File: deployed/doc-it/R17Roadmap.json [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+        "author": "PX Blue"
+      },
+      {
+        "name": "Score Card",
+        "description": "Add a component for stylized cards",
+        "year": 2020,
+        "quarter": "Q2",
+        "status": "pre-release",
+        "author": "PX Blue"
+      },
```
Comment (joebochill) [inline deployed/doc-it/R17Roadmap.json:41]: these 4 guys will be out with R16 so they can be removed from the R17 roadmap

File: deployed/doc-it/R17Roadmap.json [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+      {
+        "name": "User Menu",
+        "description": "Add support for non-clickable menu items",
+        "year": 2020,
+        "quarter": "Q2",
+        "status": "finished",
+        "author": "PX Blue"
+      },
```
Comment (joebochill) [inline deployed/doc-it/R17Roadmap.json:82]: these are done in R16 and can be removed

File: deployed/doc-it/R17Roadmap.json [hunk 3]
```diff
@@ -0,0 +1,243 @@
...
+    ],
+    "items": [
+      {
+        "name": "Drawer",
+        "description": "Add a drawer component for navigation",
+        "year": 2020,
+        "quarter": "Q2",
+        "status": "pre-release",
+        "author": "PX Blue"
+      },
```
Comment (joebochill) [inline deployed/doc-it/R17Roadmap.json:107]: this one is in r16 and can be removed

File: deployed/doc-it/R17Roadmap.json [hunk 4]
```diff
@@ -0,0 +1,243 @@
...
+      {
+        "name": "Downloadable Assets",
+        "description": "Add the ability to download designer assets through Figma",
+        "year": 2020,
+        "quarter": "Q2",
+        "status": "finished",
+        "author": "PX Blue"
+      }
```
Comment (joebochill) [inline deployed/doc-it/R17Roadmap.json:156]: this one is out now and can be removed

File: deployed/doc-it/R17Roadmap.json [hunk 5]
```diff
@@ -0,0 +1,243 @@
...
+        "author": "PX Blue"
+      },
+      {
+        "name": "Enhanced Internationalization Support",
+        "description": "Updating our Internationalization examples to support additional languages and use recommended tools",
+        "year": 2020,
+        "quarter": "Q2",
+        "status": "finished",
+        "author": "PX Blue"
+      }
```
Comment (joebochill) [inline deployed/doc-it/R17Roadmap.json:181]: this one is finished and can be removed

File: deployed/doc-it/R17Roadmap.json [hunk 6]
```diff
@@ -0,0 +1,243 @@
...
+    ],
+    "items": [
+      {
+        "name": "Simplification and Consolidation",
+        "description": "Consolidating code examples into a single repository",
+        "year": 2020,
+        "quarter": "Q2",
+        "status": "in-progress",
+        "author": "PX Blue"
+      },
```
Comment (joebochill) [inline deployed/doc-it/R17Roadmap.json:173]: Might want to split this by framework (Angular, React Native, Ionic)

File: deployed/doc-it/R17Roadmap.json [hunk 7]
```diff
@@ -0,0 +1,243 @@
...
+        "name": "React Native Login & User Registration",
+        "description": "Shared login and sign-up experience across applications",
+        "year": 2020,
+        "quarter": "Q2",
+        "status": "pre-release",
+        "author": "Innersource",
+        "applies": [
+          "react-native"
+        ]
+      },
```
Comment (joebochill) [inline deployed/doc-it/R17Roadmap.json:201]: This one is complete and can be removed

File: deployed/doc-it/R17Roadmap.json [hunk 8]
```diff
@@ -0,0 +1,243 @@
...
+    ],
+    "items": [
+      {
+        "name": "PX Blue CLI",
+        "description": "A command-line interface for starting new projects with automatic PX Blue integration",
+        "year": 2020,
+        "quarter": "Q2",
+        "status": "finished",
+        "author": "PX Blue"
+      }
```
Comment (joebochill) [inline deployed/doc-it/R17Roadmap.json:240]: The basic CLI is out, but you can update this to day something about enhancements/templates

File: deployed/doc-it/R17Roadmap.json [hunk 9]
```diff
@@ -0,0 +1,243 @@
...
+      },
+      {
+        "name": "Selection Toolbar",
+        "description": "A new component for a toolbar with dropdown selector in the subtitle",
+        "year": 2020,
+        "quarter": "Q3",
+        "status": "backlog",
+        "author": "PX Blue"
+      }
+    ]
```
Comment (joebochill) [inline deployed/doc-it/R17Roadmap.json:58]: You can probably add a placeholder row for innersourcing components from IPM

File: deployed/doc-it/R17Roadmap.json (hunk 6 above)
Comment (jeffvg) [reply deployed/doc-it/R17Roadmap.json:173]: should this be set to Q3 too

Comment (joebochill) [reply deployed/doc-it/R17Roadmap.json]: Yes, everything in R17 file should be either Q3 (majority of items) or Q4 (a couple of sneak peaks).

File: deployed/doc-it/R17Roadmap.json (hunk 9 above)
Comment (joebochill) [reply deployed/doc-it/R17Roadmap.json:26]:
```
{
                name: 'Guages and Inputs',
                description: 'Assorted input and guage components from IPM/Genepi',
                year: 2020,
                quarter: 'Q3',
                status: 'backlog',
                author: 'Innersource',
            },
```

File: deployed/doc-it/R17Roadmap.json [hunk 10]
```diff
@@ -0,0 +1,165 @@
...
+        "author": "PX Blue"
+      },
+      {
+        "name": "Selection Toolbar",
+        "description": "A new component for a toolbar with dropdown selector in the subtitle",
+        "year": 2020,
+        "quarter": "Q3",
+        "status": "backlog",
+        "author": "PX Blue"
+      }
```
Comment (joebochill) [inline deployed/doc-it/R17Roadmap.json:67]: I would replace these with "UserMenu" , "Spacer", and "ListItemTag"...I think we're probably punting on the layouts at this point until somebody starts asking about them.

File: deployed/doc-it/R17Roadmap.json [hunk 11]
```diff
@@ -0,0 +1,165 @@
...
+        "author": "PX Blue",
+        "applies": [
+          "react"
+        ]
+      },
+      {
+        "name": "Angular Login & User Registration",
+        "description": "Shared login and sign-up experience across applications",
+        "year": 2020,
+        "quarter": "Q3",
```
Comment (joebochill) [inline deployed/doc-it/R17Roadmap.json:139]: I might bump this to Q4, but it'll depend on how much we've bit off for Q3 already

File: deployed/doc-it/R17Roadmap.json (hunk 9 above)
Comment (jeffvg) [reply deployed/doc-it/R17Roadmap.json:26]: will this be for angular or react library?

File: deployed/doc-it/R17Roadmap.json (hunk 11 above)
Comment (jeffvg) [reply deployed/doc-it/R17Roadmap.json:139]: changed to Q4

**Comment by jeffvg** [comment]
updated

File: deployed/doc-it/R17Roadmap.json (hunk 9 above)
Comment (joebochill) [reply deployed/doc-it/R17Roadmap.json:26]: Angular

Comment (joebochill) [reply deployed/doc-it/R17Roadmap.json:26]: Angular

**Pull request merged**